packages/salesforcedx-vscode-automation-tests/test/specs/anInitialSuite.e2e.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { pause, Duration, log } from '@salesforce/salesforcedx-vscode-test-tools/lib/src/core/miscellaneous';
+import { ProjectShapeOption, TestReqConfig } from '@salesforce/salesforcedx-vscode-test-tools/lib/src/core/types';
+import {
+  findExtensionsInRunningExtensionsList,
+  getExtensionsToVerifyActive
+} from '@salesforce/salesforcedx-vscode-test-tools/lib/src/testing/extensionUtils';
+import { TestSetup } from '@salesforce/salesforcedx-vscode-test-tools/lib/src/testSetup';
+import { openCommandPromptWithCommand } from '@salesforce/salesforcedx-vscode-test-tools/lib/src/ui-interaction/commandPrompt';
+import {
+  getWorkbench,
+  zoom,
+  zoomReset
+} from '@salesforce/salesforcedx-vscode-test-tools/lib/src/ui-interaction/workbench';
+import { expect } from 'chai';
+import { step } from 'mocha-steps';
+import { after } from 'vscode-extension-tester';
+/*
+anInitialSuite.e2e.ts is a special case.  We want to validate that the Salesforce extensions and
+most SFDX commands are not present at start up.
+
+We also want to verify that after a project has been created, that the Salesforce extensions are loaded,
+and that the SFDX commands are present.
+
+Because of this requirement, this suite needs to run first before the other suites.  Since the
+suites run in alphabetical order, this suite has been named so it runs first.
+
+Please note that none of the other suites depend on this suite to run, it's just that if this
+suite does run, it needs to run first.
+*/
+/* eslint-disable @typescript-eslint/no-misused-promises */
+
+describe('An Initial Suite', async () => {
+  const testReqConfig: TestReqConfig = {
```

Comment:
I've not usually seen async describe blocks.  the before/after stuff might be async, and the tests might be.

Is this something special for e2e tests?